jmclient/jmclient/blockchaininterface.py

```diff
@@ -289,7 +412,7 @@ def import_addresses(self, addr_list, wallet_name, restart_cb=None):
                 jmprint(fatal_msg, "important")
             sys.exit(EXIT_FAILURE)
 
-    def import_addresses_if_needed(self, addresses, wallet_name):
+    def import_addresses_if_needed(self, addresses: List[str], wallet_name: str) -> bool:
```

Comment:
`addresses` should be a `set`, we do `addresses.issubset(imported_addresses)` (and a subtraction) below which AFAIK doesn't work with lists.

Reply:
You're right, addressed in c51f3497acbe64aca6a25afa49cd90e756fa531a.